install.sh

```diff
@@ -42,6 +42,43 @@ then
   curl -s -XPOST https://us-central1-tensorleap-ops3.cloudfunctions.net/demo-contact-bot -H 'Content-Type: application/json' -d "{\"type\":\"install-script-update-success\",\"installId\":\"$INSTALL_ID\",\"from\":\"$INSTALLED_CHART_VERSION\",\"to\":\"$LATEST_CHART_VERSION\"}" &> /dev/null &
   echo 'Done! (note that images could still be downloading in the background...)'
 else
+
+  echo Checking docker storage and memory limits...
+
+  REQUIRED_MEMORY=6227000000
+  REQUIRED_MEMORY_PRETTY=6Gb
```

Comment:
```suggestion
  REQUIRED_MEMORY_PRETTY="${REQUIRED_MEMORY:0:1}Gb"
```

Only because you have several PRETTY variables 😸

Reply:
It's harder to read your proposed line IMO (because there is no place that say 6GB explicitly you need to understand it on your own)

At first I thought maybe deriving it the other way around, but unfortunately some values are in GB and some are in GiB which makes the whole thing even more complicated